scripts/publish-schematics-verdaccio.sh

```diff
@@ -49,6 +49,9 @@ cd ../../
 cd feature-libs/storefinder
 yarn build:schematics
 cd ../../
+cd feature-libs/qualtrics
+yarn && yarn build:schematics
```

Comment:
Please remove the `yarn &&`, as it's no longer required (you need to merge the latest develop)